steps/google-play-rollout-update/0.0.1/step.yml

```diff
@@ -0,0 +1,57 @@
+title: Google Play Rollout Update
+summary: |
+  This step connects to Google Play and check if there is a release in production with an in progress rollout. If found, it increases it to the next rollout percentage [2%, 5%, 10%, 20%, 50%, 100%]
+description: |
+  The step makes use of the python library and Edits Api from Google Play. It downloads the info from the production track, and looks if any release in there has a 'userFraction' property defined. In case it has, and it is between 0 and 1, then it just modifies it to the next step, or completes the release if next is 100%
+website: https://github.com/angelolloqui/bitrise-step-google-play-rollout-update
+source_code_url: https://github.com/angelolloqui/bitrise-step-google-play-rollout-update
+support_url: https://github.com/angelolloqui/bitrise-step-google-play-rollout-update/issues
+published_at: 2021-12-30T14:26:01.867676+01:00
+source:
+  git: https://github.com/angelolloqui/bitrise-step-google-play-rollout-update.git
+  commit: 94a8272d685732e7c1386132c9a8184076e75dc1
```

Comment:
It seems the tag has been moved since the StepLib PR, based on the step.yml version 0.0.1 refers to the commit: `94a8272d685732e7c1386132c9a8184076e75dc1`, but based on the step repository this tag is on a different commit (`0d6ccc95c63247b93555a84857b451d8d8c0f74e `).

Please do not move the version tag, once a Step share process has been started.

Reply:
You are right, the tag was wrong and I moved it instead of creating a second because it was not published yet, but I know I should not do that, sorry for the inconvenience